src/main/java/com/cloudbees/jenkins/Cleaner.java

```diff
@@ -66,7 +73,7 @@ protected void doRun() throws Exception {
             for (GHRepository repo : r.resolve()) {
                 try {
                     removeHook(repo, Trigger.all().get(DescriptorImpl.class).getHookUrl());
-                    LOGGER.fine("Removed a hook from "+r+"");
+                    LOGGER.info("Removed a hook from "+r+"");
```

Comment:
It was changed because the log relative to adding the hook already was at info level, so I think the remove hook log (opposite operation) should be at info level too.